apps/ledger-live-desktop/src/renderer/families/tron/StepReceiveFunds.jsx

```diff
@@ -0,0 +1,308 @@
+  const receiveStakingFlowConfig = useFeature("receiveStakingFlowConfigDesktop");
+  const mainAccount = account ? getMainAccount(account, parentAccount) : null;
+  invariant(account && mainAccount, "No account given");
+
+  const name = token ? token.name : getAccountName(account);
+  const initialDevice = useRef(device);
+  const address = mainAccount.freshAddress;
+  const [modalVisible, setModalVisible] = useState(false);
+
+  const hideQRCodeModal = useCallback(() => setModalVisible(false), [setModalVisible]);
+  const showQRCodeModal = useCallback(() => setModalVisible(true), [setModalVisible]);
+
+  const onVerify = useCallback(() => {
+    // if device has changed since the beginning, we need to re-entry device
+    if (device !== initialDevice.current || !isAddressVerified) {
+      transitionTo("device");
+    }
+    onChangeAddressVerified(null);
+    onResetSkip();
+  }, [device, onChangeAddressVerified, onResetSkip, transitionTo, isAddressVerified]);
+
+  const onFinishReceiveFlow = useCallback(() => {
+    const id = account?.currency?.id;
+    const dismissModal = global.localStorage.getItem(`${LOCAL_STORAGE_KEY_PREFIX}${id}`) === "true";
+    if (
+      !dismissModal &&
+      receiveStakingFlowConfig?.enabled &&
+      receiveStakingFlowConfig?.params[id]?.enabled
+    ) {
+      track("button_clicked", {
+        button: "continue",
+        page: window.location.hash
+          .split("/")
+          .filter(e => e !== "#")
+          .join("/"),
+        currency: currencyName,
+        modal: "receive",
+        account,
+      });
+
+      transitionTo("stakingFlow");
+      history.push({
+        pathname: `/account/${account.id}`,
+      });
+    } else {
+      onClose();
+    }
+  }, [
+    account,
+    currencyName,
+    history,
+    onClose,
+    receiveStakingFlowConfig?.enabled,
+    receiveStakingFlowConfig?.params,
+    transitionTo,
+  ]);
+
+  return (
+    <>
+      <Box px={2}>
+        <TrackPage
+          category={`Receive Flow${eventType ? ` (${eventType})` : ""}`}
+          name="Step 3"
+          currencyName={currencyName}
+        />
+        {verifyAddressError ? (
+          <ErrorDisplay error={verifyAddressError} onRetry={onVerify} />
+        ) : isAddressVerified === true ? (
+          // Address was confirmed on device! we display a success screen!
+
+          <Box alignItems="center">
+            <SuccessDisplay
+              title={<Trans i18nKey="receive.successTitle" />}
+              description={
+                <LinkWithExternalIcon
+                  style={{ display: "inline-flex", marginLeft: "10px" }}
+                  onClick={() => openURL(urls.recipientAddressInfo)}
+                  label={<Trans i18nKey="common.learnMore" />}
+                />
+              }
+            >
+              <Box flow={4} pt={4} horizontal justifyContent="center">
+                <Button event="Page Receive Step 3 re-verify" outlineGrey onClick={onVerify}>
```

Comment:
I guess it's an analytics button so it doesn't mean to be translated + it's a copy paste of the StepReceiveFunds file because we want to exclude Tron specific part from the generic branch